pkg/controller/appsync/helpers.go

```diff
@@ -51,6 +56,20 @@ func (c *Controller) syncApps(ctx context.Context, apps *clients.AppsResponse) *
 			continue
 		}
 
+		if _, found := agencyUpdatedAlready[realm.RegionCode]; !found {
+			// Sync the realm level items.
+			realm.AgencyBackgroundColor = app.AgencyColor
+			realm.AgencyImage = app.AgencyImage
+			if err := c.db.SaveRealm(realm, database.System); err != nil {
+				merr = multierror.Append(merr, fmt.Errorf("unable to update agency information: %w", err))
+				continue
+			}
+
+			if app.AgencyImage != "" {
```

Comment:
This feels weird to me. What behavior are we trying to achieve when a realm has multiple apps with images? It feels like we should only execute the parent `if` if `AgencyImage != "`

Reply:
I was trying to account for "removing" an agency image via sync as well

Reply:
Is removing an image something we're thinking about? I guess I'd rather handle that as a one-off thing, because I think this logic is confusing